packages/sdk/test/evm/before-setup.ts

```diff
@@ -268,6 +269,13 @@ export const mochaHooks = {
         gasSettings: {
           maxPriceInGwei: 10000,
         },
+        supportedChains: [
+          {
+            chainId: 31337,
+            rpc: ["http://localhost:8545"],
+            nativeCurrency: { name: "ETH", symbol: "ETH", decimals: 18 },
+          },
+        ],
```

Comment:
why do you need this?

Reply:
we added this for `getChainProvider` to work for the tests